python3/relocate.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script ensures that the installation's sysconfig module contains the
+# correct paths. Without this, any C modules will fail to compile.
+
+root="$1"
+cd "${root}"
```

Comment:
```suggestion
cd "${root}" || exit 1
```

```
In relocate.sh line 7:
cd "${root}"
^----------^ SC2164 (warning): Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: 
cd "${root}" || exit
```

Reply:
I applied `set -euo pipefail` so this is unnecessary.